scripts/mkcap.py

```diff
@@ -158,6 +162,18 @@ def type_kind_var(self, value):
         else:
             raise Exception('Invalid value for variable attribute type_kind_var.')
 
+    @property
+    def actions(self):
+        '''Get the action strings for the variable.'''
+        return self._actions
+
+    @actions.setter
+    def actions(self, values):
+        if type(value)==dict and 'in' in values.keys() and 'out' in values.keys():
+            self._actions = values
```

Comment:
Thanks for checking this PR thoroughly. There wouldn't be any danger having other keys, because only those two are queried ever. Nonetheless, I will change this to only accept "in" and/or "out".

Reply:
Ok, fixed that one.